test/reflection.jl

```diff
@@ -2,21 +2,40 @@
 # It's hard to really test these, but just running them should be
 # sufficient to catch segfault bugs.
 
-function test_code_reflection(freflect, f, types)
+function test_ast_reflection(freflect, f, types)
+    @test !isempty(freflect(f, types))
+end
+
+function test_bin_reflection(freflect, f, types)
     iob = IOBuffer()
     freflect(iob, f, types)
     str = takebuf_string(iob)
     @test !isempty(str)
 end
 
-println(STDERR, "The following 'Returned code...' warnings indicate normal behavior:")
-test_code_reflection(code_native, ismatch, Tuple{Regex, AbstractString})
-test_code_reflection(code_native, +, Tuple{Int, Int})
-test_code_reflection(code_native, +, Tuple{Array{Float32}, Array{Float32}})
+function test_code_reflection(freflect, f, types, tester)
+    tester(freflect, f, types)
+    tester(freflect, f, (types.parameters...))
+end
 
-test_code_reflection(code_llvm, ismatch, Tuple{Regex, AbstractString})
-test_code_reflection(code_llvm, +, Tuple{Int, Int})
-test_code_reflection(code_llvm, +, Tuple{Array{Float32}, Array{Float32}})
+macro test_code_reflections(tester, freflect)
```

Comment:
it's usually not too bad to open new bug reports, since it's often pretty quick and painless to mark them as a duplicate and close them. and it also lets devs know that it is still an issue and didn't get fixed along the way.